packages/components/src/presentation-components/tree/DataProvider.ts

```diff
@@ -127,26 +127,18 @@ export interface PresentationTreeDataProviderDataSourceEntryPoints {
  * @public
  */
 export class PresentationTreeDataProvider implements IPresentationTreeDataProvider, IDisposable {
-  private _imodel: IModelConnection;
-  private _ruleset: RulesetOrId;
   private _pagingSize?: number;
```

Comment:
I think we should remove this in favor of paging size we have in props

Reply:
Done